common-crafting.lic

```diff
@@ -78,6 +78,16 @@ module DRCC
     end
   end
 
+  def find_loom_room(hometown, override = nil)
+    if override
+      DRCT.walk_to(override)
+    else
+      loomrooms = get_data('crafting')['tailoring'][hometown]['loom-rooms']
```

Comment:
`loom_rooms`

Reply:
I just cut and pasted `def find_shaping_room` and changed `shapeingrooms` to `loomrooms`. Changed!